### PR TITLE
Issue-3-add-guests-to-list | Natalie Nunez

### PR DESCRIPTION
**Pull request by NatalieNunez**
issue-3

### CODE REVIEW

File: client/guest/guest-form.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    this.setState({
+      lastName: target.value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const firstName = this.state.firstName;
+    const lastName = this.state.lastName;
+    const form = document.getElementById('guest-form');
```
Comment (thebearingedge) [inline client/guest/guest-form.jsx]: we don't query the DOM like this in React. is `event.target` what you're looking for?

File: client/guest/guest-form.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    const newGuest = {
+      firstName,
+      lastName
+    };
+    this.props.onSubmit(newGuest);
+    form.reset();
```
Comment (thebearingedge) [inline client/guest/guest-form.jsx]: If you are using state to track values, it's better to use `setState` to reset them.

File: client/guest/guest-form.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+class GuestForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.getAllGuests = this.getAllGuests.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.state = {
+      isClicked: false,
```
Comment (thebearingedge) [inline client/guest/guest-form.jsx]: `isClicked` isn't as clear as `modalOpen`.

File: client/guest/guest-form.jsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      .then(guests => {
+        this.setState({
+          guests
+        });
+      });
+  }
+
+  componentDidMount() {
+    this.getAllGuests();
+  }
```
Comment (thebearingedge) [inline client/guest/guest-form.jsx]: Is this necessary here? i saw it in another file and wasn't too sure

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, just some minor changes.

**Comment by NatalieNunez** [comment]
Request changes fixed
removed dom query, reset form using setState, updated state names for clarity, removed repeated code

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**